update.sh

```diff
@@ -13,12 +19,11 @@ if [ ! -d openbsd ]; then
 		git clone "${RPKICLIENT_GIT}/openbsd"
 	fi
 fi
-if [ -d openbsd/.git ]; then
-	(cd openbsd/src
-	 git fetch
-	 git checkout "${openbsd_branch}"
-	 git pull --rebase)
-fi
+
+(cd openbsd
+ git fetch
+ git checkout "${openbsd_branch}"
+ git pull --rebase)
```

Comment:
Do you see a possibility to keep it conditional? At Fedora, the build system is not allowed to connect to the Internet, thus I am prefilling this directory by other means…and thus I would like to avoid the git commands.

Reply:
For these cases I think official release tarballs should be used.
Our goal is to decouple rpki-client releases from the OpenBSD release cycle and so there will be more frequent releases which should reduce the need to use this repo directly.

Reply:
Right, but I still would like to be able to create a rpki-client RPM from specific Git commits for testing purposes. It's *very handy* to be able to put the two Git commits into the RPM build process if needed. So I would be keen on that small `if`/`fi` thing :)

Reply:
You can pass a git commit to ./update.sh as argument and it will checkout that version.